### PR TITLE
feat: v1.7 migration docs

### PR DESCRIPTION
**Pull request by ShubhamPalriwala**
## What does this PR do?
Mentions about:
- Data Migration
- Formbricks/JS revamp

Also updates the `package.json` for **database** package for better readability!

### Required

- [ ] Filled out the "How to test" section in this PR
- [ ] Read [How we Code at Formbricks](<[https://github.com/formbricks/formbricks/blob/main/CONTRIBUTING.md](https://formbricks.com/docs/contributing/how-we-code)>)
- [ ] Self-reviewed my own code
- [ ] Commented on my code in hard-to-understand bits
- [ ] Ran `pnpm build`
- [ ] Checked for warnings, there are none
- [ ] Removed all `console.logs`
- [ ] Merged the latest changes from main onto my branch with `git pull origin main`
- [ ] My changes don't cause any responsiveness issues

### Appreciated

- [ ] If a UI change was made: Added a screen recording or screenshots to this PR
- [ ] Updated the Formbricks Docs if changes were necessary

### CODE REVIEW

File: apps/formbricks-com/app/docs/self-hosting/migration-guide/page.mdx [hunk 1]
```diff
@@ -8,6 +8,84 @@ export const metadata = {
 
 # Migration Guide
 
+## v1.7
+
+Formbricks v1.7 comes with a big new feature like Multi-Language Surveys and other features to help you minimize the friction in reaching out to your users. This also involves a few changes in our environment variables. This guide will help you migrate your existing Formbricks instance to v1.7 without any hassles or build errors.
```
Comment (mattinannt) [inline apps/formbricks-com/app/docs/self-hosting/migration-guide/page.mdx:13]: `This also involves a few changes in our environment variables`
What are the changes in the env variables? Or is this copy/paste from the last release?

File: apps/formbricks-com/app/docs/self-hosting/migration-guide/page.mdx [hunk 2]
```diff
@@ -8,6 +8,84 @@ export const metadata = {
...
+
+<Col>
+<CodeGroup title="Migrate the data">
+
+```bash
+docker run --rm \
+  --network=formbricks_default \
+  -e DATABASE_URL="postgresql://postgres:postgres@postgres:5432/formbricks?schema=public" \
+  -e UPGRADE_TO_VERSION="v1.7" \
+  ghcr.io/formbricks/data-migrations:v1.7
```
Comment (mattinannt) [inline apps/formbricks-com/app/docs/self-hosting/migration-guide/page.mdx:74]: The tag is wrong. Can we maybe just use the `latest` tag (omit it)?

File: apps/formbricks-com/app/docs/self-hosting/migration-guide/page.mdx [hunk 3]
```diff
@@ -8,6 +8,84 @@ export const metadata = {
...
+
+To run all these steps, please navigate to the `formbricks` folder where your `docker-compose.yml` file is located.
+
+1. **Backup your Database**: This is a crucial step. Please make sure to backup your database before proceeding with the upgrade. You can use the following command to backup your database:
+
+<Col>
+<CodeGroup title="Backup Postgres">
+
+```bash
+docker exec formbricks-quickstart-postgres-1 pg_dump -U postgres -d formbricks > formbricks_pre_v1.7_$(date +%Y%m%d_%H%M%S).sql
```
Comment (mattinannt) [inline apps/formbricks-com/app/docs/self-hosting/migration-guide/page.mdx:32]: Please check the updated 1.6 docs for the updated command

**Review by mattinannt — CHANGES_REQUESTED** [review]
@ShubhamPalriwala thanks for the migrations guide :-) 💪 There were a few changes & feedback in the meantime in the 1.6 migration guide that we should also include/fix in these docs:

File: apps/formbricks-com/app/docs/self-hosting/migration-guide/page.mdx (hunk 1 above)
Comment (ShubhamPalriwala) [reply apps/formbricks-com/app/docs/self-hosting/migration-guide/page.mdx:13]: yep i wrote that safer side beforehand if there were any, removed it for now

**Pull request closed without merging**